tests/e2e/framework/framework.go

```diff
@@ -211,3 +211,13 @@ func (f *Framework) WaitDefaultServiceAccountCreated(ns string, timeout time.Dur
 		return true, nil
```

Comment:
Unfortunately, the e2e-tests fail for me with the following output:

```
▶ make e2e-test
go test -v ./tests/e2e/... --server "" --kubeconfig "<PATH>/.kube/config"
=== RUN   TestAutomountServiceAccountToken
=== RUN   TestAutomountServiceAccountToken/defaultServiceAccountUndefinedAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount
=== RUN   TestAutomountServiceAccountToken/defaultServiceAccountchange-defaultAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount
=== RUN   TestAutomountServiceAccountToken/defaultServiceAccountchange-allAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount
=== RUN   TestAutomountServiceAccountToken/dedicatedServiceAccountUndefinedAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount
=== RUN   TestAutomountServiceAccountToken/dedicatedServiceAccountchange-defaultAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount
=== RUN   TestAutomountServiceAccountToken/dedicatedServiceAccountchange-allAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount
--- FAIL: TestAutomountServiceAccountToken (57.90s)
    --- FAIL: TestAutomountServiceAccountToken/defaultServiceAccountUndefinedAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount (0.25s)
        admission_automount_token_test.go:139: expected is mounted to be false but is true with [{default-token-p9hrg {nil nil nil nil nil &SecretVolumeSource{SecretName:default-token-p9hrg,Items:[],DefaultMode:*420,Optional:nil,} nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil}}]
    --- FAIL: TestAutomountServiceAccountToken/defaultServiceAccountchange-defaultAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount (0.23s)
        admission_automount_token_test.go:139: expected is mounted to be false but is true with [{default-token-qz8j4 {nil nil nil nil nil &SecretVolumeSource{SecretName:default-token-qz8j4,Items:[],DefaultMode:*420,Optional:nil,} nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil}}]
    --- FAIL: TestAutomountServiceAccountToken/defaultServiceAccountchange-allAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount (0.26s)
        admission_automount_token_test.go:139: expected is mounted to be false but is true with [{default-token-4wb9x {nil nil nil nil nil &SecretVolumeSource{SecretName:default-token-4wb9x,Items:[],DefaultMode:*420,Optional:nil,} nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil}}]
    --- PASS: TestAutomountServiceAccountToken/dedicatedServiceAccountUndefinedAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount (49.08s)
    --- PASS: TestAutomountServiceAccountToken/dedicatedServiceAccountchange-defaultAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount (7.03s)
    --- FAIL: TestAutomountServiceAccountToken/dedicatedServiceAccountchange-allAnnotationUndefinedServiceAccountAutomountUndefinedPodAutomount (1.06s)
        admission_automount_token_test.go:139: expected is mounted to be false but is true with [{dedicated-token-qdwsm {nil nil nil nil nil &SecretVolumeSource{SecretName:dedicated-token-qdwsm,Items:[],DefaultMode:*420,Optional:nil,} nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil nil}}]
=== RUN   TestAutomountServiceAccountTokenInDefaultNamespace
--- PASS: TestAutomountServiceAccountTokenInDefaultNamespace (6.03s)
    admission_automount_token_test.go:185: expected is mounted to be false but is true
=== RUN   TestAutomountServiceAccountTokenEditServiceAccount
--- FAIL: TestAutomountServiceAccountTokenEditServiceAccount (0.03s)
    admission_automount_token_test.go:229: expected updated service account to have automoutnServiceAccountToken set to false but is nil
=== RUN   TestAutomountServiceAccountTokenDefaultServiceAccountFromConfig
--- FAIL: TestAutomountServiceAccountTokenDefaultServiceAccountFromConfig (0.21s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1c77c06]

[...]

FAIL	github.com/karydia/karydia/tests/e2e	64.223s
?   	github.com/karydia/karydia/tests/e2e/framework	[no test files]
make: *** [e2e-test] Error 1
```

Reply:
`kubectl logs` of the karydia pod revealed that karydia is not able to start because of the following error and this leads to the wrong (-> because karydia is not running) K8s cluster behaviour:
```
Failed to load KarydiaDefaultNetworkPolicy : karydianetworkpolicies.karydia.gardener.cloud "karydia-default-network-policy" not found
```

Reply:
You were right, this was causing the issue. The name of the default network policy was wrong and karydia couldn't find it. Should work now, thank you!